fiasco/ions.py

```diff
@@ -56,9 +57,13 @@ def __init__(self, ion_name, temperature: u.K, *args, **kwargs):
         # TODO: do not hardcode defaults, pull from rc file
         self._dset_names = {}
         self._dset_names['ioneq_filename'] = kwargs.get('ioneq_filename', 'chianti')
-        self._dset_names['abundance_filename'] = kwargs.get('abundance_filename',
-                                                            'sun_coronal_1992_feldman_ext')
         self._dset_names['ip_filename'] = kwargs.get('ip_filename', 'chianti')
+        abund = kwargs.get('abundance', 'sun_coronal_1992_feldman_ext')
```

Comment:
Let's make this an explicit keyword argument rather than sticking it in `kwargs` (this is my fault, but we may as well change it here while we're mucking with the abundance!).